### DISCUSSION TITLE
Which tools were used to create the documentation?

### DISCUSSION THREAD

**Discussion by KuSi833** [SilentVoid13/Templater#361] [Help] [answered]
I really appreciate the effort put into creating the docs, and am wondering which tools were used.

**Comment by caseyg** [comment, marked as answer]
@KuSi833 Found the answer, here:

https://github.com/SilentVoid13/Templater/blob/gh-pages/404.html#L6

> Docusaurus v2.0.0-alpha.72

https://docusaurus.io

**Reply by KuSi833** [reply]
Thank you!
